Leave/leave_interface.py

```diff
@@ -132,3 +132,16 @@ def AddRetroactiveLeaveToDB(member, message_id, start_date, end_date, leave_type
 
         leave_db.InsertLeave(member.id, message_id, leave_type, day, reason, "", leave_status, is_emergency, is_unpaid)
     return ("Retroactive leave was inserted successfully")
+
+def IsMemberOnLeave(member_id, date):
+    work_days = utils.GetWorkDays(date, date)
+    if len(work_days) <= 0:
+        return (True, "Holiday/Weekend")
```

Comment:
Weekends shouldn't count as leaves. 

We could have another function `isWorking(date)` to handle this case if we need to

Reply:
True, but it is basically the same output as the current implementation, because in the future when we try to add the logic for counting a public holiday also we will need to ad it to the new function as well which causes more moving parts

Reply:
Done